packages/core/src/utils/create/index.js

```
@@ -279,129 +279,135 @@
+      });
+
+      serviceConfig.config = {
+          ...serviceConfig.config,
+          priority: {
+              ...(serviceConfig.config.priority ? serviceConfig.config.priority : {}),
+              postRequest: 10,
+              postRequestError: 10,
+          },
+      };
+      // 开启 postRequest 和 postRequestError
+      serviceConfig.config.postRequest = true;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)